test/integration/GCHP/integrationTest.sh

```diff
@@ -178,6 +193,15 @@ fi
 logsDir="${itRoot}/${LOGS_DIR}"
 scriptsDir="${itRoot}/${SCRIPTS_DIR}"
 
+# If --no-bootstrap is selected, also remove the lines in
+# integrationTestExecute that reset EFYO -> CYS and EFY -> CYS
+if [[ "x${bootStrap}" == "xno" ]]; then
+    sed_ie '/HEMCO_Config.rc  \# GC_RESTART$/d' \
+	   "${scriptsDir}/integrationTestExecute.sh"
+    sed_ie '/HEMCO_Config.rc  \# GC_BCs$/d' \
+	   "${scriptsDir}/integrationTestExecute.sh"
+fi
```

Comment:
This won't have any effect since we don't use restart settings in HEMCO_Config.rc for GCHP. The line to change is [here](https://github.com/geoschem/geos-chem/blob/542bb8fad21bd117404a5b5e27a06627dc858676/test/shared/commonFunctionsForTests.sh#L184).